src/Controls/src/Core/Label/Label.cs

```diff
@@ -357,18 +352,19 @@ void Span_GestureRecognizer_CollectionChanged(object sender, NotifyCollectionCha
 
 		void ITextAlignmentElement.OnHorizontalTextAlignmentPropertyChanged(TextAlignment oldValue, TextAlignment newValue)
 		{
+			// This is a no-op since the horizontal text alignment does not affect bounds or
+			// any other property that would require a measure invalidation.
 		}
 
 		static void OnTextPropertyChanged(BindableObject bindable, object oldvalue, object newvalue)
 		{
 			var label = (Label)bindable;
-			LineBreakMode breakMode = label.LineBreakMode;
-			bool isVerticallyFixed = (label.Constraint & LayoutConstraint.VerticallyFixed) != 0;
-			bool isSingleLine = !(breakMode == LineBreakMode.CharacterWrap || breakMode == LineBreakMode.WordWrap);
-			if (!isVerticallyFixed || !isSingleLine)
-				((Label)bindable).InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
```

Comment:
This is a "major" change where I think the calls will prevent a valid measure from taking place. 

Assume you have a label:
```cs
var label = new Label {
  HeightRequest = 100,
  MinimumHeightRequest = 10,
  LineBreakMode = LineBreakMode.NoWrap,
}
```

This condition will evaluate as follows:

```cs
bool isVerticallyFixed = true; // the constraint evals to fixed as there is a height constraint
bool isSingleLine = true; // the break mode is NoWrap and thus is NOT NOT wrapping
if (!isVerticallyFixed || !isSingleLine) // becomes if (!true || !true) or false
```

However, if you look at the label definition, it is not a wrapping label and it has a fixed height. However, it does NOT have a fixed with. This means updating the text WILL require a measure since the width has changed. But here we do not call it.